agent_build_refactored/managed_packages/install-scriptlets/preuninstall.sh

```diff
@@ -43,4 +43,10 @@ fi
 # Always remove the .pyc files and __pycache__ directories
 find /usr/share/scalyr-agent-2 -type f -name '*.py[co]' -delete -o -type d -name __pycache__ -exec rm -r {} \;
 
+# Remove dynamically generated venv.
+rm -r "/var/opt/scalyr-agent-2/venv"
+
+# Collect garbage after Python.
+find /opt/scalyr-agent-2 -type f -name '*.py[co]' -delete -o -type d -name __pycache__ -exec rm -r {} \;
```

Comment:
Will the `*.py[co]` delete be necessary with Python 3?  (I though Python 3 puts everything in `__pycache__`)

Reply:
Ah, I just reused existing code snipped. Good to know, Will remove that. Thanks.